src/hook/pauseResume.js

```diff
@@ -0,0 +1,299 @@
+/**
+* Copyright (c) 2015 NAVER Corp.
+* egjs projects are licensed under the MIT license
+*/
+eg.module("pauseResume", ["jQuery"], function($) {
+	"use strict";
+
+	var animateFn = $.fn.animate;
+	var stopFn = $.fn.stop;
+	var uuid = 1;
+
+	function now() {
+		return new Date().getTime();
+	}
+
+	function AniProperty(prop, optall, prevProp, isFirst) {
+		this.opt = optall;
+		this.start = -1;
+		this.elapsed = 0;
+		this.paused = false;
+		this.uuid = uuid++;
+		this.easingNames = [];
+		this.prop = isFirst ? {} : $.extend({}, prevProp);
+
+		for (var propName in prop) {
```

Comment:
The prop is inline style object. So If element is computed style I think that not working.
Right?